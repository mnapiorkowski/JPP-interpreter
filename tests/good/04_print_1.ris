void main():
    let int a = 42;
    let bool b = true;
    let string c = "foo"

    print a;
    print b;
    print c;
    print "\n"

    println a;
    println b;
    println c;

    return;