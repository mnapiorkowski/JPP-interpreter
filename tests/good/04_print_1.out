42truefoo
42
true
foo
